sandbox/apparmor/apparmor.go

```diff
@@ -496,9 +496,15 @@ func AppArmorParser() (cmd *exec.Cmd, internal bool, err error) {
 	for _, dir := range filepath.SplitList(parserSearchPath) {
 		path := filepath.Join(dir, "apparmor_parser")
 		if _, err := os.Stat(path); err == nil {
-			// Pin 4.0 abi if host supports it.
+			// Detect but ignore apparmor 4.0 ABI support.
```

Comment:
Captured task to revisit this after switching to AppArmor 4.0: https://warthogs.atlassian.net/browse/SNAPDENG-18643